### PR TITLE
WEN-28:  Create filter component, also admin page front

### PR DESCRIPTION
**Pull request by viliusmac**
admin page reachable by:  localhost:4200/admin

### CODE REVIEW

File: flower-e-shop-frontend/src/app/components/filters/filters.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(FiltersComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+/*  it('should create', () => {
+    expect(component).toBeTruthy();
+  });*/
```
Comment (AivarasRinkevicius) [inline flower-e-shop-frontend/src/app/components/filters/filters.component.spec.ts:24]: Do we need this test for the future?

File: flower-e-shop-frontend/src/app/components/login/login.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(LoginComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+/*  it('should create', () => {
+    expect(component).toBeTruthy();
+  });*/
```
Comment (AivarasRinkevicius) [inline flower-e-shop-frontend/src/app/components/login/login.component.spec.ts:24]: Same here

**Review by AivarasRinkevicius — APPROVED** [review]

**Review by JuozasVainauskas — APPROVED** [review]
Great!🥇

File: flower-e-shop-frontend/src/app/components/filters/filters.component.spec.ts (hunk 1 above)
Comment (viliusmac) [reply flower-e-shop-frontend/src/app/components/filters/filters.component.spec.ts:24]: we might

**Pull request merged**